src/components/Pagination/PaginationContainer/PaginationContainer.js

```diff
@@ -16,46 +16,66 @@ const Container = styled('div')`
 Container.displayName = 'Container';
 
 const NavigationButtonPrevious = styled(Button)`
-  border-top-right-radius: 0;
-  border-bottom-right-radius: 0;
-  border-right: 0;
+  ${({ theme }) => `
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+    border-right: 0;
+    &:hover {
+      box-shadow: 1px 0 0 ${theme.colors.n500};
+      z-index: 1;
+    }
+`};
 `;
 NavigationButtonPrevious.displayName = 'NavigationButtonPrevious';
 
 const NavigationButtonNext = styled(Button)`
   border-top-left-radius: 0;
   border-bottom-left-radius: 0;
 `;
-
 NavigationButtonNext.displayName = 'NavigationButtonNext';
 
+const PaginationFooter = styled('div')`
+  ${({ theme }) => `
+    text-align: right;
+    color: ${theme.colors.n500};
+    font-size: ${theme.typography.text.kilo.fontSize};
+    padding-right: 12px;
```

Comment:
This could be one of our spacing variables, right?

https://circuit.sumup.com/#/styles/spacing

Reply:
Thanks for the catch 👍